code-challenges.js

```diff
@@ -2,6 +2,17 @@
 
 
 // --------------------1) Create a function that returns the first 10 numbers of the Fibonacci sequence in an array. Expected output: [1, 1, 2, 3, 5, 8, 13, 21, 34, 55]
+var i; 
+var fib = [];
+
+fib [0] = 0;
+fib [1] = 1;
+for (i = 2; i <= 10; i++) {
+    fib [i] = fib [i - 2] + fib[i-1];
+    console.log(fib[i]);
+}
+// Googled this one but I sort of understand the logic!
```

Comment:
googling is ok if you understand what's happening! hopefully the assessment review helped :)